framework/src/utils/FormattedTable.C

```diff
@@ -248,19 +248,15 @@ FormattedTable::printTable(std::ostream & out,
     term_width = getTermWidth(false);
   else
   {
-    try
-    {
-      // convert the name into a number, without this conversion the id is used which
-      // is assigned automatically and is not related to the name
-      term_width = std::stoi(suggested_term_width);
-    }
-    catch (const std::invalid_argument &)
-    {
+    if (std::string(suggested_term_width).find_first_not_of("0123456789") != std::string::npos)
```

Comment:
Nope! This incorrectly passes an empty string which will fail in stoi below.

Robust solution here: https://github.com/idaholab/moose/blob/devel/framework/src/mesh/MooseMesh.C#L996-L1009

Reply:
Ugh... That's not the cleanest example. What you need to watch for is this condition:
```
// If the `>>` operator on the string stream returns false or if the `eof` bit is not set on the stream, something went wrong.
if (!(ss >> id) || !ss.eof())
```